source/block-and-log-streamer.ts

```diff
@@ -51,12 +60,19 @@ export class BlockAndLogStreamer<TBlock extends Block, TLog extends Log> {
 		this.getLogs = getLogs;
 		if (onError === undefined) throw new Error(`onError must be provided`);
 		this.onError = onError;
-		this.blockRetention = (configuration && configuration.blockRetention) ? configuration.blockRetention : 100;
+		this.blockRetention = configuration && configuration.blockRetention ? configuration.blockRetention : 100;
```

Comment:
I find it easier to read a ternary when the conditional is wrapped in parenthesis if the conditional is anything more than a single variable.  The reason for this is because most people don't memorize their order of operations beyond the basic arithmetic operations and assignment, so I prefer parenthesis anytime I have operators in a sequence.

Reply:
Agreed.